.stylelintrc

```diff
@@ -2,6 +2,7 @@
 	"plugins": ["@signal-noise/stylelint-scales"],
 	"rules": {
 		"scales/font-weight": [[ 400, 600, 700, "normal", "bold", "inherit" ]],
+		"scales/font-size": [[ 0.75, 0.875, 1, 1.25, 1.5, 2, 2.25, 3, 3.375 ], { unit: "rem" } ],
```

Comment:
Are we dropping all support for px? We have a lot of existing usages, so folks will not be able to commit without a --no-verify if folks are modifying something else in an scss file (for example updating text color in a file that already has a font-size usage with px). I'd recommend updating existing usages first before enforcing this in a precommit check.

Reply:
> Are we dropping all support for px?

Yes. Ideally we should switch to using the font variables defined in our type scale, but if not, all of those evaluate to rems.

> I'd recommend updating existing usages first before enforcing this in a precommit check.

Good call, this should wait until the remaining PRs have also shipped: #44322, #44472, #44471, #44470 -- they're small, though; the large values aren't as common, thankfully.

After that, the majority of px values should have been converted to vars (although I've noticed a few 18px creeping in...not sure if I missed them or if they came after the conversions).

Reply:
Sounds good for landing order! I think we may want to give folks a heads up to the change (especially since folks might not be as familiar with REM) in a post before landing the lint check but it's a not a huge deal if some new ones creep in. We can always tidy up afterward.